fastseq_cli/transformers_generate.py

```diff
@@ -225,8 +250,21 @@ def generate_summaries_or_translations(
                 data_queue.close()
                 msg_queue.close()
                 sys.exit(1)
-            summaries_cpu = summaries.cpu()
-            data_queue.put((ind, summaries_cpu))
+            scores = summaries.scores
+            sequences = summaries.sequences
+            scores_cpu = None
+            if output_sequence_scores:
+                if (type(summaries) == BeamSearchEncoderDecoderOutput or \
+                    type(summaries) == BeamSearchDecoderOnlyOutput or \
+                    type(summaries) == BeamSampleDecoderOnlyOutput or \
+                    type(summaries) == BeamSampleEncoderDecoderOutput):
```

Comment:
nit: `if type(summaries) in [BeamSearchEncoderDecoderOutput, ..., BeamSampleEncoderDecoderOutput]`?